.evergreen.yml

```diff
@@ -14,13 +17,13 @@ functions:
       type: setup
       params:
         directory: src/github.com/mongodb/mongodb-enterprise-kubernetes
-
+        
   "install goreleaser":
     - command: shell.exec
       type: setup
-      include_expansions_in_env:
-        - goreleaser_pro_tar_gz
```

Comment:
was this a bug?

Reply:
I think I ran into an issue because of it, but I don't exactly remember.
As these expansions were all under "params" in the other functions, I moved it for consistency

Reply:
Do we use that env var at all? Might be an left over we might be able to remove

Reply:
Removed